src/SDKs/DataLake.Store/Management.DataLake.Store/generate.ps1

```diff
@@ -0,0 +1 @@
+powershell.exe -ExecutionPolicy Bypass -NoLogo -NonInteractive -NoProfile -File "$(split-path $SCRIPT:MyInvocation.MyCommand.Path -parent)\..\..\..\..\tools\generateTool.ps1" -ResourceProvider "datalake-store/resource-manager" -PowershellInvoker  -AutoRestVersion "latest"
```

Comment:
@lewu-msft please model the generate.ps1 according to what you see over here
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/generate.ps1#L1

Reply:
@shahabhijeet Thx! updated